docs/learn/learn-guides-polkadot-opengov.md

```diff
@@ -66,16 +73,30 @@ have been met. Failing to submit the decision deposit within a
 {{ polkadot: <RPC network="polkadot" path="const.referenda.undecidingTimeout" defaultValue={201600} filter="blocksToDays"/> :polkadot }}{{ kusama: <RPC network="kusama" path="const.referenda.undecidingTimeout" defaultValue={201600} filter="blocksToDays"/> :kusama }}-day
 period will lead to a referendum timeout.
 
-## Request Submission and Decision Deposit Back
+## Claiming the Preimage and Decision Deposits
+
+After a referendum finishes its life cycle (and gets approved or rejected or timed out), the
+preimage and decision deposits can be claimed. For claiming the preimage deposit, navigate to
+[Polkadot-JS UI > Governance > Preimages](https://polkadot.js.org/apps/#/preimages) and click on
+unnote button shown on the preimage you submitted.
+
+![Claim Preimage Deposit](../assets/claim-preimage-deposit.png)
+
+Similarly, to claim the decision deposit, navigate to
+[Polkadot-JS UI > Governance > Referenda](https://polkadot.js.org/apps/#/referenda) and scroll down
+to the end of the page to click on the referenda with the decision deposit and claim it.
+
+![Claim Referendum Deposits](../assets/claim-referendum-decision-deposit.png)
+
+## Claiming the Submission Deposit
```

Comment:
We mentioned about submission deposit for preimages in the earlier info note "Preimage Submission Deposit". Let us avoid ambiguity.

```suggestion
## Claiming the Referendum Submission Deposit
```